### PR TITLE
List and navigation

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my PR. Emphasize more on lists and navigation component first.

### CODE REVIEW

**Review by DivyanshSareen — APPROVED** [review]
Looks good to merge

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    border: 1px solid var(--DARK-GREY);
+}
+
+.nav-search-icon{
+    padding: 0 0 0 1rem;
+    color: var(--DARK-GREY);
+}
+
+.nav-search{
+    min-width: 0;
```
Comment (amartya-rs) [inline components/navigation/navigation.css]: why min-width: 0 ?

Comment (bhushannaikaroskar) [reply components/navigation/navigation.css]: Thanks for correcting. Was debugging something and hence tried this. Removed it

**Review by itsmebipul00 — APPROVED** [review]

File: components/navigation/navigation.css (hunk 1 above)
Comment (amartya-rs) [reply components/navigation/navigation.css]: Cool

**Review by amartya-rs — APPROVED** [review]

**Pull request merged**